extensions/positron-assistant/src/commands/default.ts

```diff
@@ -128,17 +137,10 @@ export async function defaultHandler(
 
 	// When invoked from the editor, add selection context and editor tool
 	if (request.location2 instanceof vscode.ChatRequestEditorData) {
+		system += await fs.promises.readFile(`${mdDir}/prompts/chat/editor.md`, 'utf8');
 		const document = request.location2.document;
 		const selection = request.location2.selection;
 		const selectedText = document.getText(selection);
-		const hasSelection = selection && !selection.isEmpty;
```

Comment:
Also just realized why this is needed: when editing via an inline chat, it should only edit the selection.